frontend/lib/app-context.tsx

```diff
@@ -168,6 +168,13 @@ export interface AppServerInfo {
    * intended for training or review purposes.
    */
   isDemoDeployment: boolean;
+
+  /**
+   * An optional "Facebook App ID" for the an associated organization's Facebook page.
+   * If empty (the default), there will be no reference to a Facebook App ID
+   * in the header metatags for the site.
+   */
+  facebookAppId?: string;
```

Comment:
Based on the code it looks like this will never actually be `undefined`, so maybe remove the `?`?  That way we'll be sure that it is either an empty string or a populated string, rather than possibly undefined, or an empty string, or a populated one (and less cases == less edge cases == less future headaches lol).  If you do that though, TypeScript will likely complain that the `FakeServerInfo` object in `frontend/lib/tests/util.tsx` doesn't have an entry for it, so you'll want to set it to the empty string there!

Reply:
Noice! All set